packages/block-library/src/cover/edit/index.js

```diff
@@ -151,8 +151,9 @@ function CoverEdit( {
 	const setMedia = attributesFromMedia( setAttributes, dimRatio );
 
 	const onSelectMedia = async ( newMedia ) => {
+		const newUrl = newMedia?.type === 'image' ? newMedia.url : undefined;
```

Comment:
If we don't have a test to cover videos, should we add a small comment here to explain why we're only using any image URL?

Reply:
Done.